app/coffeescripts/views/courses/roster/EditEnrollmentsView.coffee

```diff
@@ -0,0 +1,64 @@
+#
+# Copyright (C) 2012 - present Instructure, Inc.
+#
+# This file is part of Canvas.
+#
+# Canvas is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, version 3 of the License.
+#
+# Canvas is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+define [
+  'i18n!course_settings'
+  'jquery'
+  'underscore'
+  'compiled/views/DialogBaseView'
+  'compiled/views/courses/roster/RosterDialogMixin'
+  'jst/courses/roster/EditEnrollmentsView'
+  'str/htmlEscape'
+  'compiled/jquery.rails_flash_notifications'
+  'jquery.disableWhileLoading'
+], (I18n, $, _, DialogBaseView, RosterDialogMixin, editEnrollmentsViewTemplate, h) ->
+
+  class EditEnrollmentsView extends DialogBaseView
+
+    @mixin RosterDialogMixin
+
+    dialogOptions:
+      id: 'edit_sections'
+      title: I18n.t 'titles.edit_enrollment', 'Edit Enrollment'
+
+    render: ->
+      @$el.html editEnrollmentsViewTemplate
+        context_modules: @model
+      this
+
+    update: (e) =>
+      e.preventDefault()
```

Comment:
This needs JS confirmation telling user that they should really be sure of what they're doing as there isn't an easy un/re-do, only proceed if they agree